cmd/kubeadm/app/master/manifests_test.go

```diff
@@ -460,14 +462,10 @@ func TestGetAPIServerCommand(t *testing.T) {
 
 	for _, rt := range tests {
 		actual := getAPIServerCommand(rt.cfg, false)
-		for i := range actual {
-			if actual[i] != rt.expected[i] {
-				t.Errorf(
-					"failed getAPIServerCommand:\n\texpected: %s\n\t  actual: %s",
-					rt.expected[i],
-					actual[i],
-				)
-			}
+		sort.Strings(actual)
```

Comment:
Many thanks for fixing this. Has been annoying but haven't got to fixing it :+1: